analyzer/codechecker_analyzer/cmd/analyze.py

```diff
@@ -819,6 +819,11 @@ def main(args):
     if not os.path.exists(args.output_path):
         os.makedirs(args.output_path)
 
+    # TODO: I'm not sure that this directory should be created here.
+    fixit_dir = os.path.join(args.output_path, 'fixit')
+    if not os.path.exists(fixit_dir):
+        os.makedirs(fixit_dir)
```

Comment:
Did you check that we will not store this directory on CodeChecker store command?

Reply:
Yes, I did. CodeChecker store command has a white list basically which tells what to include in the .zip file.